### PR TITLE
test that shows how

### PR DESCRIPTION
**Pull request by darkdarkdragon**
"Offer total cannot be negative" error appears

### CODE REVIEW

File: test/amount-test-error.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    var a2 = {
+        "currency": "JPY",
+        "issuer": "r94s8px6kSw1uZ1MV98dhSRTvc6VMPoPcN",
+        "value": "66435.49665972557"
+    };
+    var a1a = Amount.from_json(a1);
+    var res = a1a.add(a2).subtract(a2).subtract(a1);
+
+    console.log(res.to_human());
+    assert(!res.is_negative(), 'Offer total cannot be negative');
```
Comment (sublimator) [inline test/amount-test-error.js]: You're saying it's tiny fractional amounts ?

Comment (darkdarkdragon) [inline test/amount-test-error.js]: Yep. I've pushed new commit that shows events that lead to the error.

Comment (sublimator) [inline test/amount-test-error.js]: Good catch

Comment (darkdarkdragon) [inline test/amount-test-error.js]: Thanks. It was a long hunt :smile:

Comment (clark800) [inline test/amount-test-error.js]: Awesome job finding a test case to reproduce it!

File: test/amount-test-error.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+/* eslint-disable max-len */
```
Comment (clark800) [inline test/amount-test-error.js]: Seems like this is unnecessary

**Comment by clark800** [comment]
@darkdarkdragon Can we fix up the lint errors and add a skip for the unit test so we can get this merged?

**Comment by clark800** [comment]
And squash please

**Comment by darkdarkdragon** [comment]
ok

**Comment by darkdarkdragon** [comment]
@clark800 done

**Comment by clark800** [comment]
LGTM

**Pull request merged**